packages/gatsby/src/commands/develop.js

```diff
@@ -201,7 +219,7 @@ async function startServer(program, { activity }) {
 
   app.use(
     require(`webpack-dev-middleware`)(compiler, {
-      logLevel: `trace`,
+      logLevel: `silent`,
```

Comment:
why do we set this to silent?

Reply:
to get rid of `wdm` terminal output as we handle webpack errors/warnings, recompiling/done ourselves with structured logging stuff